docs/platform/howto/saml/add-okta-idp.md

```diff
@@ -66,6 +70,13 @@ Login failed. Please contact your account administrator for more details.
 
 Ensure **IdP initiated login** is enabled.
 
+### Invalid relay state
+
+If you get the invalid relay state error, then you are attempting an
+IdP-initiated auth flow. This happens, for example, when you click the
+Aiven SAML app in Okta. Set the **Default Relay State** in Okta to the
+Aiven console that your organization uses.
```

Comment:
You might want to capitalize "console".

Reply:
It's the general noun here, not the proper noun (there are multiple consoles).